MachUpskillingFY17/JabbR-Core
# Justine/twitter ms auth

To Test:
- First you need to set up usersecrets for Facebook, Google, Twitter, and Microsoft
    - If you tested social auth previously, you should already have the usersecrets for FB and Google, however the appid/ secret has changed for Google so you will need to delete your current one and add the new one
    - Email me for the secrets 

- Then you need to set up localhost to resolve to jabbrcore.net
    - In your c:\windows\system32\drivers\etc\ folder open the hosts file in Notepad as an administrator
    - Add the line `127.0.0.1 jabbrcore.net` to the end of the file and save it

- Now we are ready to test social auth
    - Open Visual Studio as an administrator
    - Run the project *without* debugging (Ctrl + F5)
    - You should be able to sign in with all four social auth types and create rooms/ send messages in those rooms as well as all other current functionality 

- When you are done testing make sure to delete the line you added to your c:\windows\system32\drivers\etc\hosts file
